src/luaotfload-colors.lua

```diff
@@ -197,6 +199,20 @@ local function node_colorize (head, toplevel, current_color, current_transparent
             n_replace, current_color, current_transparent = node_colorize(n_replace, false, current_color, current_transparent)
             setdisc(n, n_pre, n_post, n_replace)
 
+        elseif n_id == glue_t then
+            if getsubtype(n) >= 100 then
+                local n_leader = getleader(n)
```

Comment:
I think it might be nicer to check n_leader ~= nil instead if subtype >= 100: This avoids the magic number 100 and saves a function call.

Reply:
Yes indeed. But this part of the code should be removed now.